pkg/services/authz/server.go

```diff
@@ -44,6 +44,11 @@ func newLegacyServer(
 	return s, nil
 }
 
+func (s *legacyServer) AuthFuncOverride(ctx context.Context, fullMethodName string) (context.Context, error) {
```

Comment:
Claudiu I'm removing that change as we are going to change the authenticator and we'll be able to set no access token there.